### ISSUE TITLE
Pretrained VQ-GAN on 64x64 Bair dataset

### ISSUE THREAD

**Issue by julesberman** [Araachie/river#6]
Hi! Thanks for the great work and quality code.

I am trying to use the pre-trained VQ-GAN on 64x64 Bair dataset. I use all the default settings you have in this repo, but I am finding the after passing the video through the GAN the FVD against the original video is quite high. ~270. 

I was wondering if this is your experience or if there is something I might be missing?

Thanks any help would be very appreciated!

**Comment by Araachie** [comment]
Hi!

Thanks for the interest in our work!

I don't remember the exact number, but it was definitely lower than the FVD of the generated videos vs the autoencoded videos, which is 73.5 (see Table 2 in the paper), and the reconstruction FID was around 6. How do the autoencoded videos look visually? I recall that on the frame-level they were good, although some temporal inconsistency was present.

**Comment by julesberman** [comment]
Thanks for the response.

So, for me, on the frame level, the auto encoded look significantly worse. The output of the VQ-GAN is pretty far outside the [0,1] range and it results low quality images (even after renormalization). Overall, its clearly worse than it should be (as reported in the paper)

My theory is that there is some step I am missing when the data is prepared as input to the VQ-GAN (either normalization or augmentation of some sort, or something when it is downloaded) or there is option for the VQ-GAN that I need to turn on (or off) like quantization or something like this.

I know it's probably difficult to debug without seeing my code, but I used your code exactly where ever possible. This is why I think it is some data preparation issue. 

If you have anything thoughts it would be greatly appreciated.
